setup.py

```diff
@@ -15,23 +15,28 @@
 import os
 from setuptools import setup, find_packages
 from typing import List, Dict, Tuple
+from utils.artifacts import get_release_and_version
 
-# default variables to be overwritten by the version.py file
-is_release = None
-version = "unknown"
-version_major_minor = version
 
-# load and overwrite version and release info from compressed-tensors package
-exec(open(os.path.join("src", "compressed_tensors", "version.py")).read())
-version_nm_deps = f"{version_major_minor}.0"
+package_path = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), "src", "compressed_tensors"
+)
+(
+    is_release,
+    version,
+    version_major,
+    version_minor,
+    version_bug,
+) = get_release_and_version(package_path)
+
+version_nm_deps = f"{version}.0"
```

Comment:
I think we should keep the old code for version_nm_deps as f"{version_major_minor}.0" otherwise installing its deps will fail